Assets/AltTester/Runtime/UI/AltDialog.cs

```diff
@@ -492,15 +501,23 @@ private void onDisconnect(int code, string reason)
                     setMessage(reason, ERROR_COLOR, true);
                 });
             }
+            else if (code == 1001)
+                updateQueue.ScheduleResponse(() =>
+                    {
+                        responseCode = 0;
+                        stopClients();
+                        setInteractibilityForRestartButton(false);
+                    });
             else
             {
-                if (code == 1001)
-                    updateQueue.ScheduleResponse(() =>
-                        {
-                            responseCode = 0;
-                            stopClients();
-                            setInteractibilityForRestartButton(false);
-                        });
+                updateQueue.ScheduleResponse(() =>
+                {
+                    responseCode = 0;
+                    if (wasConnected)
+                        setInteractibilityForRestartButton(false);
+                    stopClients();
+
+                });
```

Comment:
You can refactor this to be something like this
```
 else
            {
                updateQueue.ScheduleResponse(() =>
                {
                    responseCode = 0;
                    if (wasConnected||code == 1001)
                        setInteractibilityForRestartButton(false);
                    stopClients();
                });
```
this way you don't need two else